integration-test/src/test/java/org/sagebionetworks/ITDrsControllerTest.java

```diff
@@ -34,4 +114,36 @@ public void testGETDrsServiceInfo() throws SynapseException {
         assertEquals(serviceInformation.getType().getArtifact(), "drs");
         assertEquals(serviceInformation.getType().getVersion(), "1.2.0");
     }
+
+    @Test
+    public void testGETDrsObjectBlob() throws SynapseException {
```

Comment:
This test is only passing because the user that is getting the drs object is the same user that created it.  Will need an additional user to test this.